osc/message.go

```diff
@@ -196,11 +205,11 @@ func (msg *Message) ParseMessage() error {
 	//     to make the tag portion of a length divisible by 4
 	//     If already divisible by 4, there will be 4 null bytes
 	if err != nil {
-		return fmt.Errorf("No null byte following tags")
+		return errors.New("no null byte following tags")
 	}
 
 	// Trim off the zero byte we just wrote to msg.Tags
-	msg.Tags = strings.TrimSuffix(msg.Tags, string(0))
+	msg.Tags = strings.TrimSuffix(msg.Tags, "0")
```

Comment:
The intention is to trim the zero byte, not the ascii character "0".  To remove the warning we encounter when calling string(0), refer to the zero byte as string(rune(0))